Apps/phawscloudtrail/awscloudtrail.json

```diff
@@ -273,9 +272,8 @@
             "description": "Lookup the management events captured by CloudTrail",
             "parameters": {
                 "attribute_key": {
-                    "description": "Select an Attribute to query on (or blank)",
+                    "description": "Select an Attribute to query by (or blank) which to search",
```

Comment:
remove 'which to search' from this description. Also, what does '(or blank)' mean here? I would describe that as well, something like '(leave blank if ...)'

Reply:
Yes, we have changed the description like **"Select an Attribute to query by (or leave blank for retrieve all records)"**